mama/c_cpp/src/cpp/MamaDQPublisherManager.cpp

```diff
@@ -316,6 +317,11 @@ void MamaDQPublisherManager::destroyPublisher (const char *symbol)
 	mImpl->destroyPublisher (symbol);
 }
 
+MamaDQPublisher* MamaDQPublisherManager::getPublisher (const char* symbol)
+{
+    return mImpl->getPublisher (symbol);
```

Comment:
Only if allocation of it failed in the constructor (in which case create() would have failed too).  In general in MAMA we assume we are dealing with a non-NULL impl in individual functions.

The rest are all good spots though, we'll resubmit taking them on board.